knex/knex.d.ts

```diff
@@ -7,7 +7,9 @@
 /// <reference path="../node/node.d.ts" />
 
 declare module "knex" {
-  import Promise = require("bluebird");
+  if (!Promise) {
```

Comment:
Statement is not allowed in `.d.ts` context.
You can't change importing module flexible...